interview-questions.md

```diff
@@ -4,49 +4,49 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. What does CRUD stand for?
 
-  Your answer:
+  Your answer: Create, Read, Update ,Delete.
 
-  Researched answer:
+  Researched answer: CRUD is an acronym that stands for Create, Read, Update, and Delete. These are the four most basic operations that can be performed with most traditional database systems and they are the backbone for interacting with any database. 
 
 
 
 2. What are the 5 HTTP verbs?
 
-  Your answer:
+  Your answer: post, get, put, patch, delete
 
-  Researched answer:
+  Researched answer: The primary or most-commonly-used HTTP verbs (or methods, as they are properly called) are POST, GET, PUT, PATCH, and DELETE.
 
 
 
 3. When creating a basic (stateless) class component in React, what are the necessary elements needed to render "Hello World" on the page?
 
-  Your answer:
+  Your answer: Return & <h1,p,footer etc...><.../>, and if you have multiple elements, a <div> tag to wrap around it.
 
-  Researched answer:
+  Researched answer: A functional(a.k.a. stateless) component is just a plain javascript function which takes props as an argument and returns a react element.
```

Comment:
this question was a little confusing and your answer is great. but just so you know, we were looking for: import, class name extends, render, return, div would be a bonus, export included in this answer somehow